packages/bits/package.json

```diff
@@ -89,25 +89,17 @@
   },
   "analyze": false,
   "peerDependencies": {
-    "@angular-devkit/schematics": "^11.1.4",
     "@angular/animations": "^11.1.2",
     "@angular/cdk": "^11.1.1",
-    "@angular/common": "^11.1.2",
```

Comment:
common and compiler are a natural dependency of `forms` and are included in all Angular apps.